articles/ghostty_pane_splitter.md

```
@@ -35,46 +44,65 @@ cargo install ghostty-pane-splitter
+- `new_split:down`
+- `goto_split:next`
+- `goto_split:previous`
+- `equalize_splits`
+
+キーの組み合わせは任意で構いません。設定の詳細は [Ghostty のキーバインドリファレンス](https://ghostty.org/docs/config/keybind/reference) を参照してください。以下は設定例です。
 
-```bash
-git clone https://github.com/rikeda71/ghostty-pane-splitter.git
-cd ghostty-pane-splitter
-cargo install --path .
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)